include/help_util.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HELP_UTIL_H
+#define HELP_UTIL_H
+
+#include <map>
+#include <string>
+
+namespace HelpUtil {
+
+enum class Filter { ALL, COMMON };
+
+enum class Category { MISC, FILE, DOSBOX, BATCH };
+
+enum class CmdType { SHELL, PROGRAM };
```

Comment:
UpperCamelCase for enum values?

Reply:
Suggest making then all `: uint8_t`. 
Might as well carry around 3 bytes instead of 24 bytes per program.